attributes/default.rb

```diff
@@ -1,10 +1,12 @@
+default['mailcatcher']['ruby_version'] = '1.9.3-p545'
+
 default['mailcatcher']['http_ip'] = '0.0.0.0'
 default['mailcatcher']['http_port'] = 1080
 
 default['mailcatcher']['smtp_ip'] = '127.0.0.1'
 default['mailcatcher']['smtp_port'] = 1025
 
-default['mailcatcher']['bin'] = 'mailcatcher'
+default['mailcatcher']['bin'] = '/opt/rbenv/shims/mailcatcher'
```

Comment:
I see. I think it happens because rbenv install cannot update environment variables inside running shells. So already executed bash processes won't see the updated PATH variable. Even if you do an `export PATH=...` that makes the variable available to sub-shells only. That's why the startup init script don't know about the mailcatcher binary, because it still holds the old PATH variable which does not contain the reference to rbenv shims.
Can you please try to add the line `source /etc/profile` near to the beginning of the file `templates/default/mailcatcher.init.conf.erb`.